mips/malta.c

```diff
@@ -172,7 +172,7 @@ void platform_init(int argc, char **argv, char **envp, unsigned memsize) {
 
   setup_kenv(argc, argv, envp);
 
-  uart_init();
+  cn_init();
```

Comment:
You're right, but we'll take care of it later, when more than one console device is handled. Corresponding function in FreeBSD is [cnadd](http://bxr.su/FreeBSD/sys/kern/kern_cons.c#168). For starter we'll leave `tty0` and `tty1` for use by user-space application.